docs/manual/mkdocs.yml

```diff
@@ -227,15 +226,62 @@ nav:
       - project:
         - project upgrade: 'cmd/spfx/project/project-upgrade.md'
     - Microsoft Graph (graph):
+      - schemaextension:
+        - schemaextension add: 'cmd/graph/schemaextension/schemaextension-add.md'
+    - Outlook:
```

Comment:
We should add the command prefix here `(outlook)`

Reply:
Done